sw/airborne/firmwares/rotorcraft/main_ap.c

```diff
@@ -305,60 +305,58 @@ void failsafe_check(void)
 {
 #if !USE_GENERATED_AUTOPILOT
   if (radio_control.status == RC_REALLY_LOST &&
-      autopilot_mode != AP_MODE_KILL &&
-      autopilot_mode != AP_MODE_HOME &&
-      autopilot_mode != AP_MODE_FAILSAFE &&
-      autopilot_mode != AP_MODE_NAV &&
-      autopilot_mode != AP_MODE_MODULE &&
-      autopilot_mode != AP_MODE_FLIP &&
-      autopilot_mode != AP_MODE_GUIDED) {
+      autopilot_get_mode() != AP_MODE_KILL &&
```

Comment:
@gautierhattenberger, yes, that was indeed my worry, mode change with unintended effect. But since you also testfly a lot we both will discover and inconsistency soon ;-) . Thanks for the code integration, really appreciated :1st_place_medal: